CancelledPage.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+import { Font } from 'expo';
+
+export default class App extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble': require('./assets/fonts/Double_Bubble_shadow.otf'),
+      'bubble-body': require('./assets/fonts/Bubbleboddy-FatTrial.ttf'),
+      'source-sans': require('./assets/fonts/source-sans-pro.semibold.ttf')
+    });
+    this.setState({ fontLoaded: true }) ;
+  }
+  render(){
+    return(
+      <View style = { styles.container }>
+        <View style = { styles.topContainer }>
+          <Text style = { this.state.fontLoaded ? styles.sorryText : styles.anything }>
```

Comment:
`styles.anything` isn't defined. Either remove it or define the style

Reply:
fixed.